docs/root/start/sandboxes/tls-inspector.rst

```diff
@@ -0,0 +1,109 @@
+.. _install_sandboxes_tls_inspector:
+
+TLS Inspector Listener Filter
+=============================
+
+.. sidebar:: Requirements
+
+   .. include:: _include/docker-env-setup-link.rst
+
+   :ref:`curl <start_sandboxes_setup_curl>`
+	Used to make ``HTTP`` requests.
+
+   :ref:`jq <start_sandboxes_setup_jq>`
+	Parse ``json`` output from the upstream echo servers.
+
+This example demonstrates how the ``TLS``  inspector can be used to select ``FilterChains`` to
+distribute the traffic between upstream clusters according to the matched ``transport_protocol`` and/or
+``application_protocols``.
+
+It also demonstrates the admin statistics generated by the ``TLS`` inspector listener filter.
+
+Step 1: Build the sandbox
+*************************
+
+Change directory to ``examples/tls-inspector`` in the Envoy repository, and bring up the services.
+
+This starts one proxy listening on ``localhost:10000``, and with an admin interface listening on port 12345.
+
+It also starts three upstream ``HTTP`` services that echo back received headers in ``json`` format.
+
+The first 2 services are ``HTTPS`` services listening on port ``443`` and the other has no ``TLS`` and listens on
+port ``80``.
+
+.. code-block:: console
+
+  $ pwd
+  envoy/examples/tls-inspector
+  $ docker-compose pull
+  $ docker-compose up --build -d
+  $ docker-compose ps
+
+                Name                               Command               State                         Ports
+  ---------------------------------------------------------------------------------------------------------------------------------
+  tls-inspector_service-http_1            docker-entrypoint.sh node  ... Up
+  tls-inspector_service-https-http1.1_1   docker-entrypoint.sh node  ... Up
+  tls-inspector_service-https-http2_1     docker-entrypoint.sh node  ... Up
+  tls-inspector_tls-inspector_1           /docker-entrypoint.sh /usr ... Up      0.0.0.0:10000->10000/tcp, 0.0.0.0:12345->12345/tcp
+
+
+Step 2: Access services
+***********************
+
+Querying the service at port ``10000`` with different ``HTTP`` version specified over ``TLS``, or
+with pure ``HTTP`` protocol. your could see the requests are handled by different services.
```

Comment:
im not sure what pure means in this context - can i suggest for this sentence

> Querying the service at port ``10000`` with a different ``HTTP`` version specified over ``TLS``, or
with ``HTTP`` protocol without ``TLS``, the requests will be handled by different upstream services.